src/api/mailbox/gmail.ts

```diff
@@ -59,7 +60,7 @@ async function getNewToken(
     const code = req.query.code as string;
     tokens = (await oAuth2Client.getToken(code)).tokens;
     oAuth2Client.setCredentials(tokens);
-    res.send("Authentication successful! You can close this window now.");
+    res.sendFile(path.join(process.cwd(), "src/api/mailbox/success.html"));
```

Comment:
This line hardcodes the file path using relative path. When releasing the app, this would need changes.